docs/learn/documentation/versioned/jobs/configuration-table.html

```diff
@@ -231,20 +231,19 @@ <h1>Samza Configuration Reference</h1>
                     <td class="property" id="job-coordinator-monitor-partition-change">job.coordinator.<br />monitor-partition-change</td>
                     <td class="default">false</td>
                     <td class="description">
-                        If you are using Kafka for coordinator stream, this configuration enables the Job Coordinator to
-                        detect partition count difference in Kafka input topics. On detection, it updates a Gauge
-                        metric of format <span class="system">system-name</span>.<span class="stream">stream-name</span>.partitionCount,
-                        which indicates the difference in the partition count from the initial state. Please note that currently this
-                        feature only works for Kafka-based systems.
+                        This is deprecated in favor of <a href="#job-coordinator-monitor-partition-change-frequency-ms">job.coordinator.monitor-partition-change.frequency.ms</a>.
                     </td>
                 </tr>
 
                 <tr>
                     <td class="property" id="job-coordinator-monitor-partition-change-frequency-ms">job.coordinator.<br />monitor-partition-change.frequency.ms</td>
                     <td class="default">300000</td>
                     <td class="description">
-                        The frequency at which the input streams' partition count change should be detected. This check
-                        can be tuned to be pretty low as partition increase is not a common event.
+                        The frequency at which the input streams' partition count change should be detected. When the input
+                        partition count change is detected, Samza will automatically restart a stateless job or fail a
+                        stateful job. This check can be tuned to be pretty low as partition increase is not a common event.
```

Comment:
"This check can be tuned to be pretty low as partition increase is not a common event."

Do we need this? (It's slightly confusing as a lower value of this field would mean aggressive partition monitoring)

Reply:
Make sense. Does the following reads better?

"A longer time interval value is recommended for jobs w/ large number of input system stream partitions, since gathering partition count may incur measurable overhead to the job."

Reply:
+1, reads better.